### ISSUE TITLE
Document how to use fields for navigation properties

### ISSUE THREAD

**Issue by ajcvickers** [aspnet/EntityFramework.Docs#699]
See https://github.com/aspnet/EntityFrameworkCore/pull/10939 and 
https://github.com/aspnet/EntityFrameworkCore/issues/11992

**Comment by KshitizGIT** [comment]
Does this ticket imply that only documentation part is remaining and the actual functionality is already in EFCore. If so, could you demonstrate some quick codes how to achieve this?

**Comment by AndriySvyryd** [comment]
@KshitizGIT Yes, if you want to use field-only navigations you can configure them like this:
```C#
b.HasMany(typeof(PostNavFields), "_posts").WithOne("_blog");
```

**Comment by ajcvickers** [comment]
See also #775
